packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -740,8 +740,17 @@ describe("Error Discovery", () => {
         assert(wrappedError.getTelemetryProperties().untrustedOrigin === 1); // But it should still say untrustedOrigin
```

Comment:
Would be nice to have a test case like `assert(!isExternalError(new LoggingError("some FF error"));

Reply:
Sounds good. I have added the changes in commit #a1d49538.